### PR TITLE
Jonmv/shut down feed client take 2

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs or @hmusum please review.

### CODE REVIEW

**Review by hmusum — DISMISSED** [review]

File: vespa-feed-client/src/test/java/ai/vespa/feed/client/impl/HttpRequestStrategyTest.java [hunk 1]
```diff
@@ -211,7 +212,7 @@ void testRetries() throws ExecutionException, InterruptedException, IOException
         assertEquals(0, stats.since(stats).bytesSent());
     }
 
-    @Test
+    @RepeatedTest(1000)
```
Comment (bjorncs) [inline vespa-feed-client/src/test/java/ai/vespa/feed/client/impl/HttpRequestStrategyTest.java:215]: Was this change intentional?

**Review by bjorncs — DISMISSED** [review]

File: vespa-feed-client/src/test/java/ai/vespa/feed/client/impl/HttpRequestStrategyTest.java (hunk 1 above)
Comment (jonmv) [reply vespa-feed-client/src/test/java/ai/vespa/feed/client/impl/HttpRequestStrategyTest.java:215]: Nope.

**Review by hmusum — APPROVED** [review]
